discovery-infra/test_infra/tools/assets.py

```diff
@@ -47,3 +65,83 @@ class NetworkAssets(Assets):
 
     def __init__(self):
         super().__init__(assets_file=consts.TF_NETWORK_POOL_PATH)
+
+
+class DynamicNetworkAssets(NetworkAssets):
```

Comment:
Why adding new class and not editing `NetworkAssets` class?

Reply:
+1